main.go

```diff
@@ -36,9 +37,11 @@ func main() {
 
 	router := mux.NewRouter().StrictSlash(true)
 
+	router.HandleFunc("/mentor/oauth", controllers.MentorOAuth).Methods("POST")
 	router.HandleFunc("/mentor", controllers.MentorReg).Methods("POST")
 	router.HandleFunc("/project", controllers.ProjectReg).Methods("POST")
 	router.HandleFunc("/project/all", controllers.ProjectGet).Methods("GET")
 
-	log.Fatal(http.ListenAndServe(":"+port, router))
+	log.Fatal(http.ListenAndServe(":"+port, handlers.CORS()(router)))
```

Comment:
Addressed the CORS issue in #12 . So, after that is merged. Resolve the merge conflict and then merge it.